projects/wmf_default.yaml

```diff
@@ -52,12 +52,13 @@ paths:
               x-modules:
                 - path: v1/content.yaml
                 - path: v1/mobileapps.yaml
-                  options: '{{options.mobileapps}}'
+                  options: '{{merge({"response_cache_control": options.purged_cache_control}, 
+                                options.mobileapps)}}'
                 - path: v1/graphoid.yaml
                   options: '{{options.graphoid}}'
                 - path: v1/summary.js
                   options:
-                    response_cache-control: 'max-age: 3600, s-maxage: 3600'
+                    response_cache_control: '{{options.purged_cache_control}}'
```

Comment:
I was considering to generally change it to `purged_cache_control`, as some end points won't have purging.